tests/test_enterprise/api/test_views.py

```diff
@@ -624,7 +624,7 @@ def test_enterprise_customer_courses(
                     'catalog': 1, 'active': True, 'enable_data_sharing_consent': True,
                     'enforce_data_sharing_consent': 'at_enrollment',
                     'branding_configuration': None, 'enterprise_customer_entitlements': [],
-                    'enable_audit_enrollment': False,
+                    'enable_audit_enrollment': False, 'identity_provider': None,
```

Comment:
We should also have a test to verify that the correct IdP provider_id is returned when the `EnterpriseCustomer` has an associated IdP.

Reply:
Done